subt/docker/robotika/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM robotika/subt-base:2020-05-18
+FROM robotika/subt-base:2020-05-26
```

Comment:
we should be careful with changing base image in more than in one active branch, i.e. I am not sure if this already contains OpenGL changes? Github branch update does not necessarily change the already deployed image ... but you are surely aware of this and at the moment you are the only "builder & base image deployer"

Reply:
In which other branch are changing the base image? I am aware of no such branch.

As for branch checkout (I assume that is what you meant, otherwise please clarify for me what do you mean by "github branch update") followed by the build of the robotika image does the expected thing (builds the robotika image given the specific base image version). The built image can be pushed to cloudsim. There are no inconsistencies. This all is the benefit of not using ":latest" but a specific version.

Reply:
Ok, as communicated in other channels - if the base image was changed in two parallel branches then merge to use one or the other wouldn't work. We would need a new build of the base image containing the combined changes.